pkgs/tools/system/htop/default.nix

```diff
@@ -18,15 +22,25 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-MwtsvdPHcUdegsYj9NGyded5XJQxXri1IM1j4gef1Xk=";
   };
 
-  nativeBuildInputs = [ autoreconfHook ];
+  nativeBuildInputs = [ autoreconfHook ]
+    ++ lib.optional stdenv.isLinux pkg-config
+  ;
 
   buildInputs = [ ncurses ]
     ++ lib.optional stdenv.isDarwin IOKit
+    ++ lib.optionals stdenv.isLinux [ libcap libnl ]
     ++ lib.optional sensorsSupport lm_sensors
     ++ lib.optional systemdSupport systemd
   ;
 
   configureFlags = [ "--enable-unicode" "--sysconfdir=/etc" ]
+    ++ lib.optionals stdenv.isLinux [
+      "--enable-affinity"
+      "--enable-capabilities"
+      "--enable-delayacct"
+    ]
+    ++ lib.optional openvzSupport "--enable-openvz"
```

Comment:
Do we need this? I don't think we have openvz support in nixos.

Reply:
I was not sure myself. If it's not supported then it can go.

Reply:
For what it's worth, support is enabled in libvirt: https://github.com/NixOS/nixpkgs/blob/1c18d9a5fbb064e8b948b75b420e3c3d78d16935/pkgs/development/libraries/libvirt/default.nix#L313

Reply:
OpenVZ support kernel 3.1 or redhat 6. https://wiki.openvz.org/Releases

No need to enable by default.

Reply:
Yeah, but we do not have any kernel with support for it (and I don't think there is any kernel with openvz support that still receives security updates), so might as well disable it in libvirt as well.

Reply:
Do we want to axe it altogether then, or just off by default?

Reply:
Just drop it. More code == more complexity.